modules/op-using-ssh-authentication-in-git-type-tasks.adoc

```diff
@@ -0,0 +1,12 @@
+// This module is included in the following assembly:
+//
+// *openshift-docs/cicd/pipelines/authenticating-pipelines-using-git-secret.adoc
+
+[id="op-using-ssh-authentication-in-git-type-tasks_{context}"]
+= Using SSH authentication in git type tasks
```

Comment:
If this is a conceptual topic, we might consider changing it like this:
```suggestion
= Introduction to using SSH authentication in Git-type tasks
```
Otherwise, if this is a procedural topic, consider using the more standard format of a procedure module.

Reply:
It's mostly a procedural topic, so retaining the title.